### PR TITLE
[Quick fix] Update book skeleton as to which chapters are complete/in progress

### PR DESCRIPTION
**Pull request by r-j-arnold**
### Summary

The book skeleton has a list of completed/WIP/needed chapters. Some chapter are now done/are now being worked on, so update this list.

### List of changes proposed in this PR (pull-request)

Update book skeleton as to which chapters are complete/in progress

### What should a reviewer concentrate their feedback on?

Have I made any stupid mistakes?

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [humans.md](humans.md).

### CODE REVIEW

File: book_skeleton.md [hunk 1]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
 
 #### Completed Chapters
 - Version Control - @r-j-arnold
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Version Control](chapters/version_control.md) - @r-j-arnold
```

File: book_skeleton.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
...
+- Open Science (Open Access, Open Source, Open Data) - @r-j-arnold
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Open Research](chapters/open_research.md) - @r-j-arnold
```

File: book_skeleton.md [hunk 3]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
...
+- Reproducibility (definition and why) - @pherterich
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Reproducibility](reproducibility.md) - @pherterich
```

File: book_skeleton.md [hunk 4]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
...
 
 #### WIP
- - Open Science (Open Access, Open Source, Open Data) - @r-j-arnold
- - Reproducibility (definition and why) - @pherterich
-
-#### Needed
- - Testing for research - @r-j-arnold and @annakrystalli
+- Testing for research - @r-j-arnold and @annakrystalli
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Testing for research](https://github.com/alan-turing-institute/the-turing-way/pull/147) - @r-j-arnold and @annakrystalli
```

File: book_skeleton.md [hunk 5]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
...
 - Continuous integration - @r-j-arnold and @annakrystalli
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Continuous integration](https://github.com/alan-turing-institute/the-turing-way/pull/146) - @r-j-arnold and @annakrystalli
```

File: book_skeleton.md [hunk 6]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
...
 - Reproducible Compute Environment - @r-j-arnold and @annakrystalli
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Reproducible Compute Environment](https://github.com/alan-turing-institute/the-turing-way/pull/148) - @r-j-arnold and @annakrystalli
```

File: book_skeleton.md [hunk 7]
```diff
@@ -8,16 +8,16 @@ The aim of this file is iterate/decide on:
...
 - Research Data Management - @pherterich and @rosiehigman
```
Comment (KirstieJane) [inline book_skeleton.md]:
```suggestion
- [Research Data Management](https://github.com/alan-turing-institute/the-turing-way/pull/196) - @pherterich and @rosiehigman
```

File: book_skeleton.md (hunk 4 above)
Comment (r-j-arnold) [reply book_skeleton.md]: Going to change this to a relative link rather than a url, like the others

File: book_skeleton.md (hunk 5 above)
Comment (r-j-arnold) [reply book_skeleton.md]: Going to change this to a relative link rather than a url, like the others

File: book_skeleton.md (hunk 6 above)
Comment (r-j-arnold) [reply book_skeleton.md]: Going to change this to a relative link rather than a url, like the others

File: book_skeleton.md (hunk 7 above)
Comment (r-j-arnold) [reply book_skeleton.md]: Going to change this to a relative link rather than a url, like the others

File: book_skeleton.md (hunk 4 above)
Comment (r-j-arnold) [reply book_skeleton.md]: Can't do a relative link as the chapter's not in master, I'm an idiot. Committing the URL

**Comment by r-j-arnold** [comment]
Thanks @KirstieJane, re the names, those were already there so I kept them. I think that it'd be great to have a conversation with everyone about exactly how credit is given with regards to chapter writing. I'll leave it as it is for now and am happy to change after.

**Pull request merged**